Name typo WorldBuilder cmake

### Prerequisites

- [x] I have searched for similar issues and confirmed this is not a duplicate

### Game Version

- [x] Command & Conquer Generals
- [x] Command & Conquer Generals: Zero Hour
- [ ] Other (please specify below)

### Bug Description

when building for unix cmake errors out. cmake has names in camel case but the actual file names are all lowercase
CMake Error at GeneralsMD/Code/Tools/WorldBuilder/CMakeLists.txt:204 (target_sources):
  Cannot find source file:

    include/terrainmaterial.h


CMake Error at Generals/Code/Tools/WorldBuilder/CMakeLists.txt:232 (target_sources):
  Cannot find source file:

    include/terrainmaterial.h

### Reproduction Steps

mkdir build
cd build
cmake .. --preset unix

### Additional Context

_No response_

--- a/Generals/Code/Tools/WorldBuilder/CMakeLists.txt
+++ b/Generals/Code/Tools/WorldBuilder/CMakeLists.txt
@@ -176,7 +176,7 @@ set(WORLDBUILDER_SRC
     "include/TeamIdentity.h"
     "include/TeamReinforcement.h"
     "include/teamsdialog.h"
-    "include/terrainmaterial.h"
+    "include/TerrainMaterial.h"
     "include/TerrainModal.h"
     "include/TerrainSwatches.h"
     "include/TileTool.h"

--- a/GeneralsMD/Code/Tools/WorldBuilder/CMakeLists.txt
+++ b/GeneralsMD/Code/Tools/WorldBuilder/CMakeLists.txt
@@ -180,7 +180,7 @@ set(WORLDBUILDER_SRC
     "include/TeamIdentity.h"
     "include/TeamReinforcement.h"
     "include/teamsdialog.h"
-    "include/terrainmaterial.h"
+    "include/TerrainMaterial.h"
     "include/TerrainModal.h"
     "include/TerrainSwatches.h"
     "include/TileTool.h"

